webhook/client.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	// Signatures older than this will be rejected by ConstructEvent
+	// DefaultTolerance signatures older than this will be rejected by ConstructEvent
```

Comment:
Thanks for the fix here, but could you make this a sensible sentence? Like `DefaultTolerance is the amount of drift in time that's allowed by ConstructEven as it's validating a signature.`